Resources/translations/fr/validators.fr.xlf

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+    <body>
+      <trans-unit id="5387d26cceb54294dafe82be9b0d61d0" resname="section.validator.identifier.format">
+        <source>section.validator.identifier.format</source>
+        <target>L'identifiant doit être constitué de lettres, chiffres ou avec le préfixe '_'.</target>
```

Comment:
the English version is not super clear (*Identifier should consist of letters, numbers or '_' with letter prefix.*), but I think it should translated to something like:
*L'identifiant doit être constitué de lettres, chiffres ou '_', ce dernier étant positionné après une lettre.*